### DISCUSSION TITLE
remove files from the recents list

### DISCUSSION THREAD

**Discussion by nilnil47** [ahrm/sioyek#985] [Q&A] [answered]
Hey,
I can't found a way to remove files from the recent files view (when you press O).
Is it possible?

Thank you very much!

**Comment by ahrm** [comment, marked as answer]
Press the `delete` key when the file is highlighted.

**Reply by nilnil47** [reply]
cool, thank you very much!
it make sense that it is that way, but I am on Mac OS and I didn't know that you need to use FN + backspace as the delete key
